### ISSUE TITLE
Various issues installing with NginX and Postgres

### ISSUE THREAD

**Issue by poVoq** [movim/movim#422]
I really hope this isn't just me doing a stupid mistake, but I can't seem to get even the admin interface to work properly following the available guides. I am on an Ubuntu 16.04 VPS with NginX, Postgres & php7.0. I have a working XMPP server on the same machine, but I don't even get to the point of setting it up as my admin interface looks like this (I can log in with the standard credentials but afterwards there seems to be no really working functionality):
http://imgur.com/a/BylRi
I have the deamon running and it seems to start up fine, however contrary to the tutorials it ends with a green "Encrypted" and nothing more, while there should be some websockets message I guess.

I tried following these tutorials and the connection to the Postgres database seems to be ok:
https://github.com/movim/movim/wiki/Install-Movim
https://freedif.org/movim-the-opensource-social-network-based-on-xmpp/
https://www.cypouz.com/article/151109/installation-plate-forme-sociale-movim (as much as I understood with my little french ;) )

On strange thing I noticed is that if I go to my domain with the /movim/ subdirectory it bounces me back to "mydomain/?login" and forgets about the movim. Directly putting in mydomain/movim/?login or ?admin seems to somewhat work but without the formatting, similar to the picture above.

I also opened the 8080 port in my ufw firewall, but I guess that isn't needed as the communication only happens internally?

Anyone got an idea what is happening?

**Comment by edhelas** [comment]
Hey, thanks for the details in the ticket. Did  you launched the daemon with the full path in the URL parameter?

> php daemon.php start --url=https://name.host/movim/ --port=8080 

As an example. 
Also please come discussing on our chatroom for installation related issues movim@conference.movim.eu :)

**Comment by poVoq** [comment]
Yes, I am starting it with the full url and even tried the "--interface=my server ip" parameter.

I think one of the main issues is the URL problem I mentioned above already. I checked now with the webconsole and it seems that all queries for additonal files  (.js etc) forget about the myurl/movim/ part also, so if I try to open myurl/movim/?admin I get a bunch of errors that myurl/app/... myurl/themes/... etc are missing, which is of course true as it is under myurl/movim/apps/...

Did I miss a config parameter for the movim base url somewhere?

Edit1: contrary to the tutorials my movim directory is not in /var/www/movim/, but /var/www/html/movim/, but since the html directory is the root webspace in my nginx I doubt that makes a difference.

Edit2: I tried modifying the index.php document_root function to add a .'/movim/' but then the oops something went wrong message comes and the syslog shows that the dirname(__FILE__) does indeed get the correct subdirectory and adding it again makes it fail.

Edit3: I guess it is some sort of problem with the composer stuff? I tried looking into the composer.phar file but it is a bit too convoluted to make quickly sense off :(
